#goal definition
string frame
float32 table_height
float32 block_size
---
#result definition
geometry_msgs/PoseArray blocks
widowx_block_manipulation/BlockPoseArray colored_blocks
---
#feedback
